usaon_vta_survey/templates/response/observing_systems.html

```diff
@@ -14,7 +16,15 @@ <h3>Observing systems</h3>
     </tr>
     {% for observing_system in response.observing_systems %}
       <tr>
-        <td>{{observing_system.short_name}}</td>
+        <td>{{observing_system.short_name}}
+        <a href="#" onclick="remove('{{url_for(
+            'obs.delete_response_observing_system',
+            survey_id=survey.id,
+            response_observing_system_id=observing_system.id,
+                    )}}'
+                    )">{{render_icon('file-minus-fill', color='danger')}}
```

Comment:
Nitpick: the horizontal spacing here is a touch hard to follow. I'd suggest `prettier`, but it doesn't know how to Jinja. There is a plugin... https://github.com/davidodenwald/prettier-plugin-jinja-template

But it seems like the wrong time to mess with it. Maybe we should record the need for HTML/Jinja formatting as an issue, what do you think?